test/e2e/suite_test.go

```diff
@@ -35,13 +35,14 @@ import (
 	. "github.com/onsi/gomega"
 	"github.com/pborman/uuid"
 	"k8s.io/kubernetes/test/e2e/framework"
+	"sigs.k8s.io/azurefile-csi-driver/test/e2e/driver"
 )
 
 const kubeconfigEnvVar = "KUBECONFIG"
 
 var azurefileDriver *azurefile.Driver
 
-func init() {
```

Comment:
With go 1.13.4, there are failure due to initializations of test flags. This change to BeforSuite solves the issue.